src/lib/bootloader/none_bootloader.rb

```diff
@@ -0,0 +1,19 @@
+require "yast"
+
+Yast.import "HTML"
+
+module Bootloader
+  # Represents when bootloader want user manage itself
+  class NoneBootloader < BootloaderBase
+    def summary
```

Comment:
the best way would be abstract method, as it do not make sense in base as it would be empty. But I can add it there.